test/PoolManager.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from "chai";
+import { buildContractTestContext, ContractTestContext } from "./ContractTestContext";
+import { parseUnits, formatUnits } from "ethers/lib/utils";
+
+const oneK18Decimal = parseUnits("1000", 18);
+const twentyK18Decimal = parseUnits("20000", 18);
+
+function getFloatFromBigNum (bigNumValue, dec = 18) {
+    return parseFloat(formatUnits(bigNumValue, dec));
+}
+
+describe("PoolManager test suit", async function () {
+    let r: ContractTestContext;
+
+    before(async () => {
+        r = await buildContractTestContext();
+
+        await r.lvUSD.setMintDestination(r.coordinator.address);
+        // mint a bit lvUSD
+        await r.lvUSD.mint(twentyK18Decimal);
+
+        // Approve pool manager to access 3CRV and lvUSD funds
+        await r.external3CRV.approve(r.poolManager.address, oneK18Decimal);
+        await r.lvUSD.approve(r.poolManager.address, oneK18Decimal);
+    });
+
+    it("Should have enough funds in account and coordinator", async function () {
+        /// Sanity check for funds in account. Does not test anything in Pool Manager
+        expect(getFloatFromBigNum(await r.external3CRV.balanceOf(r.owner.address))).to.be.gt(3400);
+        expect(getFloatFromBigNum(await r.lvUSD.balanceOf(r.coordinator.address))).to.be.eq(20000);
+    });
+
+    it("Should fail to fund pool if not enough lvUSD on coordinator", async function () {
+        const tooMuchLvUSDAmount = parseUnits("200000");
+        const promise = r.poolManager.fundPoolWith3CRV(r.owner.address, tooMuchLvUSDAmount);
```

Comment:
See comment below.

I think the code should be more explicit (get the balance before and after and do the math)